StructureDefinition/Extension-Genomics-Topology.StructureDefinition.json

```diff
@@ -0,0 +1,80 @@
+{
+  "resourceType" : "StructureDefinition",
+  "id" : "genomics-body-location-qualifier",
+  "url" : "https://fhir.nhs.uk/StructureDefinition/mcode-body-location-qualifier",
+  "version" : "2.0.0",
+  "name" : "BodyLocationQualifier",
+  "title" : "Body Location Qualifier Extension",
+  "status" : "active",
```

Comment:
status: active or draft?

Reply:
These extensions have been lifted from mCODE, which are active in that IG so I've marked them as active there (doesn't really make much difference to Genomics as everything needs to be tested anyway